### PR TITLE
Github actions for CI/CD

### PR DESCRIPTION
**Pull request by sujith-mn**

### CODE REVIEW

File: .github/workflows/sync-docs.yml [hunk 1]
```diff
@@ -0,0 +1,71 @@
+env:
+    REPO_CONSOLIDATE: "devonfw-guide"
+    BUILD_USER: ${{ secrets.BUILD_USER }}
+    BUILD_USER_PASSWD: ${{ secrets.BUILD_USER_PASSWD }}
+    BUILD_USER_EMAIL:  ${{ secrets.BUILD_USER_EMAIL }}
+    EMAIL: "icsddevonfwsupport.apps2@capgemini.com"
```
Comment (maybeec) [inline .github/workflows/sync-docs.yml:6]: Please don't put the email here. Take it from the secrets like in other repositories

**Review by maybeec — CHANGES_REQUESTED** [review]

File: .github/workflows/sync-docs.yml (hunk 1 above)
Comment (sujith-mn) [reply .github/workflows/sync-docs.yml:6]: Sure.. the BUILD_USER_EMAIL will be used.

**Review by hohwille — CHANGES_REQUESTED** [review]
Seems there has been some confusion in the sync of the different approaches how to split and name the github actions. Can you please revert the last commit.

**Review by hohwille — CHANGES_REQUESTED** [review]
@sujith-mn Looks goot to me now.
However, you are mixing indendation with 4 spaces and with 2 spaces. Can you use 2 spaces indendation everywhere as YAML really relies on indendation and readability is getting bad when indedation witdth is inconsistent. Thanks. Appart from that ready to be merged.

**Review by hohwille — APPROVED** [review]
I fixed the indent so we can merge and make progress...

**Review by maybeec — APPROVED** [review]

**Pull request merged**